### PR TITLE
Compress clinic

### PR DESCRIPTION
**Pull request by Elexy**
Compress the extra files that clinic tools generate before we upload to IPFS

### CODE REVIEW

File: runner/runner.js [hunk 1]
```diff
@@ -120,11 +123,11 @@ const run = async (params) => {
       config.log.debug(`DB store: ${JSON.stringify(result)}`)
       await persistence.store(result)
     }
-    // cleanup tmpout
-    rmfr(targetDir)
   } catch (e) {
     throw e
   }
+  // cleanup tmpout
+  rmfr(targetDir)
```
Comment (mcollina) [inline runner/runner.js]: we should wait for this to complete.

File: runner/compress.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+const clinicFiles = async (path) => {
+  try {
+    let contents = await readDir(path)
+    // find the dir
+    let clinicDir = _.find(contents, async (node) => {
+      let stats = await stat(`${path}/${node}`)
+      if (stats.isDirectory()) return node
+    })
```
Comment (mcollina) [inline runner/compress.js]: I don't think this does what you think it does. Don't mix `async`  functions and lodash.

File: runner/compress.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    if (clinicDir) {
+      await _tgzDir(`${path}/${clinicDir}`, `${path}/${clinicDir}.tar.gz`)
+      await rmfr(`${path}/${clinicDir}`)
+    } else {
+      config.log.error(`No clinic files found in ${path}`)
+    }
+  } catch (e) {
+    config.log.error(e)
+    return Error(`Error reading path: ${path}`)
```
Comment (mcollina) [inline runner/compress.js]: should this just rethrow?

File: runner/test/queue.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ const levelup = require('levelup')
 const memdown = require('memdown')
 
 const stopFn = () => {
-  console.log('stopping')
+  console.log('test/quueue.js ->', 'stopping')
```
Comment (mcollina) [inline runner/test/queue.js]:
```suggestion
  console.log('test/queue.js ->', 'stopping')
```

File: runner/compress.js (hunk 2 above)
Comment (Elexy) [reply runner/compress.js]: I've refactored using a `for of` loop. The test still passed. Not sure if it's still abuse ;-)

File: runner/compress.js (hunk 3 above)
Comment (Elexy) [reply runner/compress.js]: done

**Pull request merged**